src/cycax/cycad/cycad_part.py

```diff
@@ -393,45 +396,83 @@ def export(self) -> dict:
                 list_part.append(ret)
         dict_out = {}
         dict_out["name"] = self.part_no
-        dict_out["parts"] = list_part
+        dict_out["features"] = list_part
         return dict_out
 
-    def render(self, eng: str, side: str = None):
+    def beveled_edge(self, edge_type: str, side1: str, side2: str, radius: float = None, length: float = None):
```

Comment:
Maybe we should just have a size argument - remove radius and lenth please. It gets to complicated to explain when to radius and when to length. Also if we support other styles size is suitable (Reverse fillet, notch etc.)